src/app/features/event/event-survey/event-survey-single/event-survey-single.component.ts

```diff
@@ -0,0 +1,37 @@
+import { ChangeDetectionStrategy, Component, Input, inject } from '@angular/core';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { MatCheckboxModule } from '@angular/material/checkbox';
+import { MatRadioModule } from '@angular/material/radio';
+import { EventSurvey } from '../../event.interfaces';
+import { CommonModule } from '@angular/common';
+import { NonNullableFormBuilder, ReactiveFormsModule, Validators } from '@angular/forms';
+import { MatButtonModule } from '@angular/material/button';
+
+@Component({
+  selector: 'app-event-survey-single',
```

Comment:
Skoro w komponencie wymagane jest przekazanie jakiś danych w tym przypadku `survey!: EventSurvey` to możemy zawęzić selector